benches/crypto.rs

```diff
@@ -89,6 +89,17 @@ mod ed25519_benches {
                 .map(|key| key.public().clone())
                 .collect();
 
+            let aggregate_signature =
+                BLS12381AggregateSignature::aggregate(blst_signatures.clone()).unwrap();
+
+            c.bench_with_input(
+                BenchmarkId::new("BLS12381 aggregate verification", *size),
+                &(msg.clone(), blst_public_keys.clone(), aggregate_signature),
+                |b, i| {
+                    b.iter(|| i.2.verify(&i.1, &i.0));
```

Comment:
It would look a lot nicer if you destructured the `i` triplet in the bound variable pattern:
```
|b, (msg, keys, sig)| {
```